docs/02-install.md

```diff
@@ -144,6 +146,66 @@ file in your installation artifacts directory and simply re-export it:
 
     export KUBECONFIG=/path/to/something/auth/kubeconfig
 
+# Advnaced Installations
+While the OpenShift 4 installer's purpose in life is to streamline operations
+in order to guarantee success, there are a few options that you can adjust by
+using a configuration file. Namely, you can change the default number of
```

Comment:
I think it's worth emphasizing that everything except masters can be (re-)configured with day-2 operations.  The associated install-config settings are just a convenience.

Reply:
@wking you can reconfigure masters, too, depending on what you're trying to change, no? For example, what if you wanted to change the scheduler config? That would be day1 and a "master" change...?

Reply:
> For example, what if you wanted to change the scheduler config?

I don't know how well the cluster-API provider supports these changes, maybe it does.  But I know that etcd-joining is not automatic, so anything that ends up deleting a previous instance and provisioning a replacement master is going to leave the etcd cluster degraded.  We have long-term plans to support cluster-API-created masters, but we're not there yet.

Reply:
Yeah but I think the point here is that certain aspects of master are likely to be re-configurable to some degree, so a blanket "except masters" is probably not what we should say?